.github/workflows/chronos-install-option-py38-test.yml

```diff
@@ -46,7 +46,116 @@ jobs:
+          BIGDL_ROOT: ${{ github.workspace }}
+          ANALYTICS_ZOO_ROOT: ${{ github.workspace }}
+
+  chronos-pytorch-inference-test:
+    runs-on: [ self-hosted, Gondolin, ubuntu-20.04-lts ]
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: ./.github/actions/jdk-setup-action
+      - name: Set up Maven
+        uses: ./.github/actions/maven-setup-action
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Run Chronos[pytorch,inference] test
+        shell: bash
+        run: |
+          conda remove -n chronos-py38-env -y --all
+          conda create -n chronos-py38-env -y python==3.8 setuptools==58.0.4 -c ${CONDA_CHANNEL} --override-channels
+          source activate chronos-py38-env
+          pip install pytest==5.4.1
+          apt-get update
+          apt-get install patchelf
+          pip uninstall -y bigdl-friesian bigdl-friesian-spark3 bigdl-dllib bigdl-dllib-spark3 bigdl-orca pyspark bigdl-orca-spark3 bigdl-chronos bigdl-chronos-spark3 bigdl-nano bigdl-friesian bigdl-friesian-spark3
+          pip install --pre --upgrade bigdl-chronos[pytorch,inference]
+          bash python/chronos/dev/test/run-installation-options.sh "torch and not automl and not distributed and not diff_set_all"
+          source deactivate
+          conda remove -n chronos-py38-env -y --all
+        env:
+          BIGDL_ROOT: ${{ github.workspace }}
+          ANALYTICS_ZOO_ROOT: ${{ github.workspace }}
+
+  chronos-pytorch-inference-automl-test:
+    runs-on: [ self-hosted, Gondolin, ubuntu-20.04-lts ]
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: ./.github/actions/jdk-setup-action
+      - name: Set up Maven
+        uses: ./.github/actions/maven-setup-action
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Run Chronos[pytorch,inference,automl] test
+        shell: bash
+        run: |
+          conda remove -n chronos-py38-env -y --all
+          conda create -n chronos-py38-env -y python==3.8 setuptools==58.0.4 -c ${CONDA_CHANNEL} --override-channels
+          source activate chronos-py38-env
+          pip install pytest==5.4.1
+          apt-get update
+          apt-get install patchelf
+          pip uninstall -y bigdl-friesian bigdl-friesian-spark3 bigdl-dllib bigdl-dllib-spark3 bigdl-orca pyspark bigdl-orca-spark3 bigdl-chronos bigdl-chronos-spark3 bigdl-nano bigdl-friesian bigdl-friesian-spark3
+          pip install --pre --upgrade bigdl-chronos[pytorch,inference,automl]
+          pip install SQLAlchemy==1.4.27
```

Comment:
we may open an issue to make `SQLAlchemy<=1.4.27` a dependency in our `automl` extra requirement.

Reply:
yeah, I have opened the issue https://github.com/intel-analytics/BigDL/issues/6717. We can solve it in later pr.